desk/app/metagrab.hoon

```diff
@@ -361,13 +369,19 @@
     ::  request. simply retry.
     ::
     ?:  ?=(%cancel -.res)
-      [[(fetch url) ~] this]
+      [[(fetch url urbit-agent) ~] this]
     ::
     ?>  ?=(%finished -.res)
     =*  cod  status-code.response-header.res
+    ~&  ['fetched' url cod response-header.res]
```

Comment:
These `~&`s should probably be taken back out.

Reply:
yeah